nixos/modules/services/networking/kresd.nix

```diff
@@ -39,11 +50,12 @@ in
         Directory for caches.  They are intended to survive reboots.
       '';
     };
-    interfaces = mkOption {
+    listenPlain = mkOption {
       type = with types; listOf str;
-      default = [ "::1" "127.0.0.1" ];
+      default = [ "[::1]:53" "127.0.0.1:53" ];
       description = ''
-        What addresses the server should listen on. (UDP+TCP 53)
+        What addresses the server should listen on.
```

Comment:
I think it wouldn't hurt mentioning that it also expects a port. Maybe even mentioning that in a changelog entry might be useful to some users.

Reply:
Changed to "What addresses and ports the server [...]".

Reply:
Changelog entry added.